core/authentication_api.php

```diff
@@ -209,7 +209,7 @@ function auth_auto_create_user( $p_username, $p_password ) {
 
 	if( $t_auto_create ) {
 		# attempt to create the user
-		$t_cookie_string = user_create( $p_username, md5( $p_password ) );
+		$t_cookie_string = user_create( $p_username, "" );
```

Comment:
If administrator want to switch away from LDAP you don't want to end up with a lot of users that have an open well known password.  You want such users to reset their password and continue to work.  Also it protects against an end point, e.g. a bug in API that doesn't honor LDAP and just goes directly to the MD5 hash.

One day we may also want to support a MantisBT instance that enables internal users to use LDAP and external to use normal auth.  That gives us good protection against any mistakes there as well.

Plus, it is cheap to set to a random password.